calculators/referendum_supporters_calculator.rb

```diff
@@ -110,6 +115,53 @@ def fetch
 
   private
 
+  def augment_lists_with_committees_that_raised_money(supporting_by_measure_name, opposing_by_measure_name)
+    committees_with_contributions = ActiveRecord::Base.connection.execute(<<-SQL)
+    SELECT
+      oakland_committees."Filer_ID",
+      oakland_committees."Filer_NamL",
+      oakland_committees."Ballot_Measure",
+      oakland_committees."Ballot_Measure_Election",
+      oakland_committees."Support_Or_Oppose"
+    FROM combined_contributions
+    INNER JOIN oakland_committees
+      ON oakland_committees."Filer_ID" = combined_contributions."Filer_ID"
+    WHERE "Ballot_Measure" IS NOT NULL
+    GROUP BY
+      oakland_committees."Filer_ID",
+      oakland_committees."Filer_NamL",
+      oakland_committees."Ballot_Measure",
+      oakland_committees."Ballot_Measure_Election",
+      oakland_committees."Support_Or_Oppose";
+    SQL
+    committees_with_contributions.each do |row|
+      election_name = row['Ballot_Measure_Election']
+      bal_num = row['Ballot_Measure']
+      committee = committee_from_expenditure(row)
+
+      case row['Support_Or_Oppose']
+      when 'S'
+        supporting_by_measure_name[[election_name, bal_num]] ||= {}
+        supporting_by_measure_name[[election_name, bal_num]][row['Filer_ID']] ||= {
```

Comment:
Perhaps a comment that says:
Add this committee if it is not already in the hash table.
The logic gets a bit lost in the ||= construct. I had to ponder for a bit why this worked.